desktop/core/src/desktop/api2_tests.py

```diff
@@ -699,6 +699,29 @@ def test_create(self):
         json.loads(Document2.objects.get(type='gist', uuid=gist2['uuid']).data)['statement_raw']
     )
 
+    # Create multiple gist directories and then make a new gist
```

Comment:
--> in its own method instead of making the current one too big and non single responsability

Reply:
And will avoid response3 etc names later

Reply:
Still

Reply:
Done